src/bump.ts

```diff
@@ -165,36 +145,25 @@ export async function getVersionBumpTypeAndMessages(
       }
     }
     core.debug(`Commit ${commit.sha.slice(0, 6)} is not associated with a tag`);
+    commitList.push({ message: commit.message, sha: commit.sha });
+  }
 
-    core.debug(`Examining message: ${commit.commit.message}`);
-    const msg = getMessageAsConventionalCommit(
-      commit.commit.message,
-      commit.sha,
-      config
-    );
+  // We'll relax certain rules while processing these commits; these are
+  // commits/pull request titles that (ideally) have been validated
+  // _before_ they were merged, and certain GitHub CI settings may append
+  // a reference to the PR number in merge commits.
+  const configCopy = JSON.parse(JSON.stringify(config));
+  configCopy.rules["C014"].enabled = false; // SubjectExceedsLineLengthLimit
+  configCopy.rules["C019"].enabled = false; // SubjectContainsIssueReference
```

Comment:
Do we really want to disable this rules for the subject on merge?

Reply:
> Do we really want to disable this rules for the subject on merge?

Assuming the PR title was checked during the PR, there really isn't anything that these checks (or any of them, for that matter) add.
In the bump flow, it's too late to solve any Conventional Commit compliance issues, and we're really only interested in the intent.
One could argue that we could show warnings about noncompliance to _educate_ anyone reading the logs, but I think it'd be a safe guess that very few do that when the build is green.